assets/src/scss/blocks/core-overrides/Table.scss

```diff
@@ -49,6 +53,7 @@
 
   // Table caption
   figcaption {
+    min-height: 24px; // This is needed in case there's a link in the caption, for our styles
```

Comment:
Should we use `rem` or a variable for this value?

Reply:
I think it's fine, we usually don't really use those for heights 🤔 Mostly paddings, margins, and font sizes